elements/dash-accordion/dash-accordion.e2etest.js

```diff
@@ -0,0 +1,113 @@
+const helper = require('../../tools/selenium-helper.js');
+const expect = require('chai').expect;
+const {Key, By} = require('selenium-webdriver');
+
+describe('dash-accordion', function() {
+  beforeEach(function() {
+    return this.driver.get(`${this.address}/dash-accordion_demo.html`)
+      .then(_ => helper.waitForElement(this.driver, 'dash-accordion'));
+  });
+
+  it('should focus the next heading on [arrow right]',
+    async function() {
+      const found =
+        await helper.pressKeyUntil(
+          this.driver,
+          Key.TAB,
+          `return document.activeElement.getAttribute('role') === 'heading';`
+        );
+      expect(found).to.equal(true);
+
+      await this.driver
+        .executeScript(`window.firstHeading = document.activeElement;`);
+      await this.driver.actions().sendKeys(Key.ARROW_RIGHT).perform();
+      const focusedSomeHeading = await this.driver.executeScript(`
+        return window.firstHeading !== document.activeElement &&
+          document.activeElement.getAttribute('role') === 'heading';
```

Comment:
If I'm reading this right it seems to assert that you've made it to another heading, but doesn't assert anything about the order. In other words, it could be any heading. You may want to get `firstHeading` and `expectedNextHeading` and assert `document.activeElement === expectedNextHeading`.